packages/typescript/src/components/ObjectExpression.tsx

```diff
@@ -138,3 +138,23 @@ export function ObjectProperty(props: ObjectPropertyProps) {
     </>
   );
 }
+
+export interface ObjectSpreadPropertyProps {
+  value?: Children;
+  jsValue?: unknown;
+  children?: Children;
+}
+
+export function ObjectSpreadProperty(props: ObjectSpreadPropertyProps) {
+  let value;
+  if (props.value) {
+    value = props.value;
+  } else if (Object.prototype.hasOwnProperty.call(props, "jsValue")) {
```

Comment:
Can use Object.hasOwn for this I think 😁

Reply:
ah just copied the code above, but can change both